Export data feature fails

### Description

Export data feature fails : it creates new table in cross database but does not add data to it.  fails without error messge

### DBeaver Version

Community Edition 23.0.4

### Operating System

windows

### Database and driver

postgres sql 14.0


### Steps to reproduce

right click a table.
select export data and follow the steps to copy data to another database
it starts exporting and completes.  window goes away
it creates table but does not add data to it.
sometimes it is a success.  sometimes it fails.  it is not consistent

### Additional context

_No response_

```
Caused by: org.postgresql.util.PSQLException: ERROR: missing data for column "pa_d__c"
  Where: COPY patientupdate, line 325: ""a1s16000007a0cLAAQ","","","5001600000xpBwQAAU","","","","","","","BP /, MAP <br>CVP mmHG , RA/CVP ,..."
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
	at org.postgresql.core.v3.QueryExecutorImpl.processCopyResults(QueryExecutorImpl.java:1264)
	at org.postgresql.core.v3.QueryExecutorImpl.endCopy(QueryExecutorImpl.java:1069)
	at org.postgresql.core.v3.CopyInImpl.endCopy(CopyInImpl.java:49)
	at org.postgresql.copy.CopyManager.copyIn(CopyManager.java:185)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.ext.postgresql.model.PostgreCopyLoader.finishBulkLoad(PostgreCopyLoader.java:212)
	... 10 more
```

As I can see - issue is related with the bulk loading. Did you try to uncheck the "bulk load" setting and use export without it?

![2023-05-19 13_41_39-Window](https://github.com/dbeaver/dbeaver/assets/45152336/f9ef60c6-c79e-4bc0-8398-09e1188436d7)

Also, do you have any ideas about this issue part? 
`ERROR: missing data for column "pa_d__c"`

--- a/plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreCopyLoader.java
+++ b/plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreCopyLoader.java
@@ -204,7 +204,7 @@ public class PostgreCopyLoader implements DBSDataBulkLoader, DBSDataBulkLoader.B
 
         session.getProgressMonitor().subTask("Copy into " + tableFQN);
 
-        String queryText = "COPY " + tableFQN + " FROM STDIN (FORMAT CSV)";
+        String queryText = "COPY " + tableFQN + " FROM STDIN (FORMAT CSV, ESCAPE '\\')";
 
         try {
             Object rowCount;

